src/SignalR/server/Core/src/HubConnectionContext.cs

```diff
@@ -65,7 +65,7 @@ public HubConnectionContext(ConnectionContext connectionContext, HubConnectionCo
 
             _connectionContext = connectionContext;
             _logger = loggerFactory.CreateLogger<HubConnectionContext>();
-            ConnectionAborted = _connectionAbortedTokenSource.Token;
+            ConnectionAborted = CancellationTokenSource.CreateLinkedTokenSource(_connectionAbortedTokenSource.Token, connectionContext.ConnectionClosed).Token;
```

Comment:
Why doesn't the ConnectionClosed token firing abort the HubConnectionContext?

Reply:
It does now

Reply:
Doesn't that make it redundant to link the ConnectionClosed token here?

Reply:
Ah, because now we call Abort which cancels the token :+1:

Reply:
This linking of tokens makes me unhappy, do we need it?

Reply:
Removed